docs/modules/ROOT/pages/how-tos/synthesize.adoc

```diff
@@ -0,0 +1,256 @@
+* `apiSecretRef`: the name of the secret holding the API credentials for {git-vshn-net}.
+We use the GitLab user (`lieutenant-prod`) for all tenants.
+This user's credentials are available in the secret named `vshn-gitlab` which we set as the default.
+====
+
+=== Registering a cluster
+
+. Create a new cluster in https://control.vshn.net/syn/lieutenantclusters/vshn-lieutenant-prod/_create[control.vshn.net].
+There's quite a few fields to fill out when creating a cluster.
+The list below discusses the contents for each field.
+.. "ID": Choose a unique id for the cluster.
+This field is limited to lower-case characters `a-z`, numbers `0-9` and `-`.
+The length is limited to 1-61 characters.
++
+NOTE: This field cannot be changed after creation, it's used as the Kubernetes object name for the cluster by Lieutenant.
+
+.. "Display Name": Choose a descriptive name.
+Check first on the cluster listing overview how other clusters of the Tenant are named.
+.. "Lieutenant Tenant": Choose the tenant to which this cluster belongs
+.. "Service Level (Fact: service_level)": Check with Service Manager which Service Level applies to this cluster
+.. "Cloud", "Distribution", "Region": Check folders `cloud` and `distribution` in the https://git.vshn.net/syn/commodore-defaults[Commodore defaults repo] for already available values for fields "Cloud" and "Distribution" and "Region".
+Use "Cloud" `none` for on-premise setups.
+Field "Region" is not required to be set for "Cloud" `none`.
+If the cloud, region or distribution don't exist yet in the Commodore defaults repo, you can create the following files:
+
+* For a missing cloud, create file `cloud/<cloudname>.yml` in the repo.
+  If the missing cloud has multiple regions, also create file `cloud/<cloudname>/params.yml` in the repo.
+  For multi-region clouds the file `cloud/<cloudname>.yml` file in the repo should always have exactly the following content:
++
+.<cloud>/cloudname.yml
+[source,yaml]
+----
+classes:
+  - global.cloud.<cloudname>.params
+  - global.cloud.<cloudname>.${facts:region}
+----
+
+* For a missing region, or if you're creating a new multi-region cloud, create file `cloud/<cloudname>/<regionname>.yml` in the repo.
+* For a missing distribution, create file `distribution/<distributionname>.yml` in the repo.
+
+.. "Rancher Cluster ID (Fact: rancher_cluster_id)": If it's a Rancher cluster, add the Cluster ID from the Rancher Management Server to this field
+.. "Repository Type": Leave at "auto" to have Lieutenant manage the cluster catalog repo on https://git.vshn.net
+.. "GitRepo URL": Leave empty if "Repository Type" is auto.
+   Check with the {projectsyn-ig}, if you have special requirements for the cluster catalog repo.
+
++
+[NOTE]
+====
+Not all combinations work smoothly.
+If in doubt, seek help from the {projectsyn-ig}.
+====
+
+. Add initial configuration in the `<cluster-id>.yml` file in the tenant's Project Syn configuration repository (often called the "Syn tenant repo").
```

Comment:
```suggestion
. Add initial configuration in the `<cluster-id>.yml` file in the Tenant's Project Syn configuration repository (often called the "Syn Tenant repo").
```

Reply:
I don't think there's a reason to capitalize "tenant"

Reply:
dito

Reply:
resolving this one, it's sufficient to discuss this in one comment thread.